### DISCUSSION TITLE
how about use React-Table to make DataList?

### DISCUSSION THREAD

**Discussion by winner106** [BearStudio/start-ui-web#347] [General]
> TanStack Table, Supercharge your tables or build a datagrid from scratch for TS/JS, React, Vue, Solid & Svelte while retaining 100% control over markup and styles.

[https://codesandbox.io](https://codesandbox.io/s/github/tanstack/table/tree/main/examples/react/basic?from-embed=&file=/src/main.tsx:3515-3555)

[https://tanstack.com/table/v8/docs/adapters/react-table](https://tanstack.com/table/v8/docs/adapters/react-table)

![image](https://user-images.githubusercontent.com/50519642/234809287-356d8389-864a-411a-bbb5-8c1b33f457cb.png)

**Comment by yoannfleurydev** [comment]
Hi @winner106 

We thought about that, but at the moment, we do not feel the need to put TanStack Table into 🚀 Start UI [web] as it may add complexity for "simple" Datalist. We did add TanStack Table for some of our clients' projects, and I think it could be an awesome thing to put in the [ 🚀 Start UI [web] documentation](https://github.com/BearStudio/start-ui-web-docs) under the `Guides` part.
